src/Errors/ErrorBoundary.js

```diff
@@ -0,0 +1,32 @@
+import ErrorPage from '../Errors/Errors.js';
+import React, { Component } from 'react';
+
+class ErrorBoundary extends Component {
+  constructor(error) {
+    super(error);
+    this.state = {
+      hasError: false,
+      errorMessage: '',
+    };
+  }
```

Comment:
Joey im glad you could get the error state working! I'm gonna be picking your brain about this later!